### PR TITLE
[#5] Added home page - Banner | Product catalog

### PR DESCRIPTION
**Pull request by KavyaGopal**
[Sprint 1]Added the following:
- HTML CSS and .ts changes required for adding the banner
- HTML CSS and .ts changes required for adding the product catalog
- Json is added for the products and we are parsing through them to show the products

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "no-outputs-metadata-property": true,
+    "template-banana-in-box": true,
+    "template-no-negated-async": true,
+    "use-lifecycle-interface": true,
+    "use-pipe-transform-interface": true
+  },
+  "rulesDirectory": [
+    "codelyzer"
+  ]
+}
```
Comment (ashwin181) [inline tslint.json:91]: Can you please add new line here and in rest of the places simillarly.

File: src/app/jsonData/groceries.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+[
+    {
+        "id": 1,
+        "imgSrc": "../../../assets/items/apple.png",
```
Comment (ashwin181) [inline src/app/jsonData/groceries.json:4]: Can you please add different images and update descriptions for groceries and other JSON files.

**Review by ashwin181 — CHANGES_REQUESTED** [review]

File: src/app/components/home/home.component.html [hunk 3]
```diff
@@ -0,0 +1,210 @@
+<div class="grid-container">
+  <div class="row no-gutters banner">
+    <!-- <img class="home-img" src="../../../assets/home_img1.png"> -->
```
Comment (KulkarniVaibhav) [inline src/app/components/home/home.component.html:3]: Looks like we are not using this image, can you please remove it from repository?

File: src/app/jsonData/groceries.json (hunk 2 above)
Comment (KavyaGopal) [reply src/app/jsonData/groceries.json:4]: I will do this as part of follow up PR

File: src/app/components/home/home.component.html (hunk 3 above)
Comment (KavyaGopal) [reply src/app/components/home/home.component.html]: Done

File: tslint.json (hunk 1 above)
Comment (KavyaGopal) [reply tslint.json]: Done

**Review by ashwin181 — APPROVED** [review]
Reviewed the changes and looks good. You can merge now.

**Pull request merged**